src/app/components/dashboard/dashboard-content/dashboard-content.component.scss

```diff
@@ -3,6 +3,19 @@
   background: -webkit-linear-gradient(#f5ac28, #e07e7e);
   -webkit-background-clip: text;
   -webkit-text-fill-color: transparent;
+  font-size: 26px;
+}
+.Dashboard-btn{
+  margin: 2%;
+}
+.Dashboard-btn a{
+  width: 300px;
+}
+.btn{
+  font-size: 1.2rem;
+  padding-top:.5%;
+  height: 45px;
+}
+.text-dark-black{
+  font-size: 3rem;
 }
```

Comment:
Please Give a space

Reply:
> Please Give a space

@Suryansh5545 are you taking about giving one extra line after the last line? 'coz i have already given it but i don't know why it is not getting reflected here.

Reply:
@kajol-kumari if it is not showing then edit it in github and pull the commit to your local branch

Reply:
> @Kajol-Kumari if it is not showing then edit it in github and pull the commit to your local branch

@Suryansh5545 the extra line is already in my github "dashboard_look_same_As_EvalAI" branch. It's just not getting reflected here.

Reply:
@Suryansh5545 I just tried adding two extra blank line in .scss file but it is also not getting reflected here but it's in my branch on my github. I think maybe it's just not reflected here but it is there in the .scss file.